user/pages/03.commerce2/02.developer-guide/02.install-update/01.requirements/docs.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Requirements
+taxonomy:
+    category: docs
+---
+
+ ### Requirements
+
+ -Commerce 2.x requires Drupal 8.5.0 or newer.
+
+ Unless you are building a very simple site, you will normally want to develop
+ locally, on a copy of your site that operates separately from the live site.
+
+ If you already have a web server, make sure it satisfies [Drupal 8’s requirements].
+ The recommended memory limit is 256MB or more. For local development we recommend
+ [DDEV] (Docker-based), [Drupal VM] (advanced users), or [Acquia Dev Desktop] (beginners). You will also need [Composer].
+
+
+ ### PHP requirements
+
+ Drupal Commerce requires that you have the [bcmath](http://php.net/manual/en/intro.bc.php) extension installed.
+
+ If you are using DrupalVM, add the following to your configuration (change PHP version number if needed).
+
+ ```
+ php_packages_extra:
+   - php7.1-bcmath
```

Comment:
💯

Reply:
We should link to this search for if they need help: https://github.com/geerlingguy/drupal-vm/search?q=bcmath&type=Issues

I've seen people having issues.. before it was just php-bcmath.. now php7.1-bcmath or php7.2-bcmath